packages/core/admin/admin/src/pages/MarketplacePage/components/EmptyPluginSearch/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Typography } from '@strapi/design-system/Typography';
+import { Box } from '@strapi/design-system/Box';
+import { Flex } from '@strapi/design-system/Flex';
+import { Icon } from '@strapi/design-system/Icon';
+import EmptyStateDocument from '@strapi/icons/EmptyDocuments';
+import { EmptyPluginGrid } from './EmptyPluginGrid';
+
+export const EmptyPluginSearch = ({ content }) => {
+  return (
+    <Box position="relative">
+      <EmptyPluginGrid />
+      <Box position="absolute" top={11} width="100%">
+        <Flex alignItems="center" justifyContent="center" direction="column">
+          <Icon as={EmptyStateDocument} color="" width="160px" height="88px" />
+          <Box marginTop={6}>
+            <Typography variant="delta" as="p" textColor="neutral600">
+              {content}
+            </Typography>
+          </Box>
+        </Flex>
```

Comment:
@soupette not positive I followed your advice exactly here, but I struggled to get the desired result with a stack and  started to wonder why we are even using a `<Stack />`, shouldn't that be used when you have many items that require marginTop? Here we just have one element that needs `marginTop`, so why not use a `Box`?

(cc. @remidej)

Reply:
Yes you can use a box here but it's better to use paddings instead of margins.